### ISSUE TITLE
Add mixin for preimaging

### ISSUE THREAD

**Issue by albireox** [sdss/marvin#401]
Requested in #399. This is a good an easy idea to implement. For DR15 we would need to make the preimaging public but we can cross that bridge later.

**Comment by albireox** [comment]
Looked a bit into this. To implement we need to:

- [ ] Make the current [MaNGA Preimaging](https://trac.sdss.org/browser/data/manga/mangapreim) available in the SAS.
- [ ] Modify the tree definition of `$MANGA_PREIMAGING` to point to `trunk/data`.
- [ ] Modify the `sdss_paths.ini` definitions for `mangapreimfits` and `mangapreim`. Use `%definitiondir` instead of `designgrp`.
- [ ] If we want this available for public releases, make `$MANGA_PREIMAGING` available in the dr15 SAS.
- [ ] Create a mixin that downloads the file, opens it, and returns the astropy FITS object.

**Comment by albireox** [comment]
That sounds good to me but just to be clear, with preimaging I mostly mean the FITS file that is a cutout from the NSA imaging in the griz bands. The preimaging jpeg image that is created when the preimaging is generated can be included in the `Image` class. However, there is probably very little difference between that jpeg and the cutout we already have (the one the `Image` download tools get) so it may be a bit confusing. Or maybe we should have the `Image` tool switch to use the preimaging jpeg.

**Comment by havok2063** [comment]
Is it interesting enough to include in the `Image` tool?  I could see the tool being used to interact with the full color image, and the individual griz bands fits images.  Even though they are individual bands, in essence, it's still an image to display.  The pre-imaging jpegs are tiny little things.  I think the main image could be the larger full color image cutout, but we could include the pre-imaging jpeg for completeness.  I can't recall where the Image Cutout service pulls its images from, so there may be subtle differences between the two.  I guess it really depends on how closely you want it connected to the other Marvin Tools.

**Comment by albireox** [comment]
I'm not super concerned with where it lives as long as it is connected with the Tools. I agree the preimaging is ultimately a series of band images, but they contain more information (a header, an ivar, a PSF, etc) so I think we'd need a different class, maybe a sublcass from `Image`.

Not sure what you mean with the larger full colour image. The preimage includes everything within several effective radii of the galaxy (same as the cutouts) so not sure what else we'd be showing.

**Comment by havok2063** [comment]
Yeah that could work as well.  I'm not too concerned either if we want to keep them separate.  

I mean these images
https://data.sdss.org/sas/mangawork/manga/spectro/redux/v2_4_3/8485/stack/images/1901.png
compared to these
https://svn.sdss.org/public/data/manga/mangapreim/tags/v2_3/data/D0089XX/8980/preimage-1-209232_irg.jpg

**Comment by albireox** [comment]
Ok, that makes sense. We can provide different "zoom" levels.
